website/docs/components/link/inline/partials/code/how-to-use.md

```diff
@@ -3,19 +3,19 @@
 The most basic invocation requires some content to be passed as children and either an `@href` or `@route` argument.
 
 ```handlebars
-<Hds::Link::Inline @href="...">Watch tutorial video</Hds::Link::Inline>
+Lorem <Hds::Link::Inline @href="...">ipsum dolor</Hds::Link::Inline> sit amet consectetur adipiscing elit.
```

Comment:
Was there a reason to change from more realistic text to Lorem ipsum?

To me, including the text before and after the `Link` makes it slightly less easy to understand the code at a glance and also adds extra stuff you need to delete if you want to copy and paste the code sample.

Reply:
If you are worried consumers won't understand the intended usage, then maybe include one example after the basic one explaining the proper usage and showing the `Link` within a sentence.

Reply:
Most of the changes in the code docs were based off of this [ticket](https://hashicorp.atlassian.net/browse/HDS-1883) to add more context to how the component generally would be used.

While I agree that adding the text around the link adds complexity and might require consumers to delete unnecessary content, I'm not sure how big of an issue that is based on the simplicity of the component. I would maybe expect consumers to use these examples as reference, rather than actually copying them, and including them in block content reinforces the correct usage.

Reply:
Ok, I would still lean towards including one example explaining proper usage and showing the link within other text while keeping other examples simpler (especially examples showing the different arguments/options added) but up to you.